package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "app",
```

Comment:
Why wouldn't we use a more specific name? e.g. forms-product-page

Reply:
I don't know... this is how we do it in all the other apps though... It appears that this used to be the default when running `rails new`:

https://github.com/alphagov/forms-admin/commit/f865c488eff0a3b7282c7544f7c3a01b87c1cdd9
https://github.com/rails/jsbundling-rails/blob/v1.0.2/lib/install/package.json

Reply:
I guess we could go the other way and remove these lines from our other apps; I think we'd still need a change in this repo though, the reason I raised this PR in the first place is that when I ran `npm install` with the version of npm included in our Docker containers (9.6.6) the line `"name": "app"` gets added to `package-lock.json` even if there is no `name` defined in the `package.json` file.

Reply:
Weirdly though if I run `npm install` with version 9.6.6 outside Docker then it doesn't update `package-lock.json` 🤔

Reply:
it's probably better to give it a better name and we can do that one at a time for the others (not a high priority so as and when).

Reply:
Okay, I've updated this PR to add a name!